packages/main/src/components/protected/topbar/components/ProfilePicView.jsx

```diff
@@ -1,12 +1,15 @@
 import React, { useContext } from "react";
-import { TopbarContext } from "../context/Topbar";
-import { ProfileContext } from "../context/ProfileModal";
+// import { TopbarContext } from "../context/Topbar";
```

Comment:
If the code is not being used, remove it entirely instead of commenting it out

Reply:
ive resolved this